test/k8sT/KafkaPolicies.go

```diff
@@ -127,6 +127,9 @@ var _ = Describe("K8sValidatedKafkaPolicyTest", func() {
 		appPods := helpers.GetAppPods(apps, helpers.DefaultNamespace, kubectl, "app")
 		By("Testing basic Kafka Produce and Consume")
 
+		// Wait for the broker to be up properly.
+		helpers.Sleep(5)
```

Comment:
This is very, very fragile, how does the broker indicate that it is up? Worst case we can run a metadata retrieval request or something similar and wait until it returns with a timeout to fail the test.

Reply:
This is exactly the same logic  as coded in Kafka runtime tests and the CI bash tests. I agree this is fragile. I can remove this and fix it as a separate GH issue. It was not meant to be part of this PR. I just felt that we were seeing flakiness in the Kafka k8s test so thought its a good workaround. I will undo this change